src/components/layouts/ErrorBoundary.js

```diff
@@ -16,6 +17,7 @@ class ErrorBoundaryComponent extends Component {
   render() {
     const { error } = this.state;
     if (error) {
+      Sentry.captureException(error);
```

Comment:
I'm not sure if `render()` is the best place for this. We might be sending duplicate errors since this will get called whenever the component re-renders. @geoffreydhuyvetters what do you think?

Reply:
There is indeed a better place and that is `ErrorFallback`